app/models/drupal_comment.rb

```diff
@@ -103,4 +107,29 @@ def notify(current_user)
     end 
   end
 
+  def answer_comment_notify(current_user)
```

Comment:
Again, here is a lot of repeated code. I really like this PR and am excited to merge it, but I'm worried about maintainability. Think if someone tried to change this behavior in the future; would they know to change it in both places? Even if we can't get this to inherit from another model cleanly, could we create a super class that both models inherit from, or a mixin that gives both nodes and answers access to these methods?